.github/workflows/tests.yml

```
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
+        python-version: ['3.9', '3.10', '3.11', '3.12']
```

Python 3.9 has reached it's end of life
```suggestion
        python-version: ['3.10', '3.11', '3.12']
```
Could also test 3.13 if Qiskit supports it
```suggestion
        python-version: ['3.10', '3.11', '3.12', '3.13']
```